samplePDF/FDMCStruct.h

```diff
@@ -10,7 +9,7 @@ struct fdmc_base {
   bool signal; // true if signue                                              
   int nEvents; // how many MC events are there                              
   std::string flavourName;
-  int *isbound;
+
   int **Target; // target the interaction was on
```

Comment:
there is no good reason. It is basically how it has been on T2K since before I joined. This needs to be changed and is somewhere on the list. I will open this as an issue. I would actually prefer it if we created some small MaCh3Event object and each samplePDF kept a vector of these which should be more efficient cache hit wise.